### PR TITLE
Adding testing to test_mkieee.F90

### PR DESCRIPTION
**Pull request by AidanHartnett**
part of #426

### CODE REVIEW

File: tests/test_mkieee.F90 [hunk 1]
```diff
@@ -9,9 +9,9 @@ program test_mkieee
   implicit none
 
 #if KIND == 4
-  integer, parameter :: num = 4
+  integer, parameter :: num = 5
   real :: rieee(num)
-  real :: a(num) = (/ 4.3000000, 5.6000000, 1.6700000, 2.3300000 /)
+  real :: a(num) = (/ 4.3000000, 5.6000000, 1.6700000, 2.3300000 -4.3000000 /)
```
Comment (edwardhartnett) [inline tests/test_mkieee.F90]: You need a comma.

Comment (edwardhartnett) [reply tests/test_mkieee.F90]: OK, there is another bug in this test.

Where it says:

`#if KIND == 4
`
Change it to:

`#ifdef KIND_4
`

Currently none of the code in the test is ever being compiled. ;-)

That's how you can forget a comma and not get a compile error.

Comment (edwardhartnett) [reply tests/test_mkieee.F90]: See #467

File: tests/test_mkieee.F90 [hunk 2]
```diff
@@ -29,6 +29,13 @@ program test_mkieee
      print *, i, a(i), b(i)
      if (abs(a(i) - b(i)) .gt. .00001) stop 20
   end do
+
+  call mkieee(a, rieee, num)
+
+  do i = -1, num
```
Comment (edwardhartnett) [inline tests/test_mkieee.F90]: Take lines 36-38 out.

The value of the output will be checked on line 30.

File: src/mkieee.f [hunk 3]
```diff
@@ -59,7 +59,7 @@ subroutine mkieee(a,rieee,num)
               n = n - 1
            enddo
         endif
-!        n=floor(alog(atemp)/alog2)
+        n=floor(alog(atemp)/alog2)
```
Comment (edwardhartnett) [inline src/mkieee.f]: Is this change unintentional? You should not need to change the code, just the tests.

Comment (edwardhartnett) [reply src/mkieee.f]: If you can remove this change we can merge the rest of the PR and get these tests into use...

Comment (AidanHartnett) [reply src/mkieee.f]: ok sounds good

**Review by edwardhartnett — APPROVED** [review]

**Comment by edwardhartnett** [comment]
I will merge this after #480

**Pull request merged**